scraping-task-solution/MuhammadSawalhy.py

```diff
@@ -0,0 +1,64 @@
+import requests
+import os
+from tqdm import tqdm
+from bs4 import BeautifulSoup
+
+
+gucians = []
+page_URL = 'https://commits.top/egypt.html'
+r = requests.get(page_URL)
+soup = BeautifulSoup(r.content, "html.parser")
+developers = soup.select(".users-list a")
+cache_dir = ".dev-cache"
+
+
+print("Scrapping, keep your patience!!!")
+print("--------------------------------")
+
+
+if os.path.isfile(cache_dir):
+    os.remove(cache_dir)
+if not os.path.isdir(cache_dir):
+    os.mkdir(cache_dir)
+
+
+for dev in tqdm(developers):
+    URL = dev["href"]
+    _ = URL.split("/")
```

Comment:
use _ only when you will not use it again
https://stackoverflow.com/questions/5893163/what-is-the-purpose-of-the-single-underscore-variable-in-python/5893946